File: 프로그래머스/lv1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.py
def solution(numbers):
    answer = set()
    for i in range(len(numbers)):
        for j in range(len(numbers)):
            if(i==j):
                pass
            else:
                answer.add(numbers[i]+numbers[j])
    answer = list(answer)
    answer.sort()
    return answer

numbers = [2, 1, 3, 4, 1]
print(solution(numbers))

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.java
import java.util.ArrayList;
import java.util.List;
import java.util.stream.Collectors;

class Solution {
    public  List<Integer> solution(int[] numbers) {
        List<Integer> result = new ArrayList<>();
        int temp;

        for (int i = 0; i < numbers.length - 1; i++) {
            for (int j = i + 1; j < numbers.length; j++) {
                temp = numbers[i] + numbers[j];
                if (!result.contains(temp))
                    result.add(temp);
            }
        }

        return result.stream().sorted().collect(Collectors.toList());
    }
}